scripts/apps/search/components/fields/AssociatedItemsList.tsx

```diff
@@ -14,32 +14,50 @@ interface IProps {
 }
 
 interface IState {
-    associations: Array<IArticle>;
+    types: Array<string>;
+}
+
+interface IRelated {
```

Comment:
```
interface IRelatedArticle {
    _id: IArticle['_id'];
    type: IArticle['type'];;
}
```

This would be clearer. I'm not sure if field relations are correct, so feel free to change them if they aren't.